### PR TITLE
[RFR] misc facet changes

### PR DESCRIPTION
**Pull request by ThieryMichel**

### CODE REVIEW

File: src/app/js/public/facet/AppliedFacetList.js [hunk 1]
```diff
@@ -26,18 +30,29 @@ export const AppliedFacetListComponent = ({ facets }) => (
...
+    clearAll: PropTypes.func.isRequired,
 };
 
 const mapStateToProps = state => ({
     facets: fromFacet.getAppliedFacetList(state),
 });
 
-export default connect(mapStateToProps)(AppliedFacetListComponent);
+const mapDispatchToProps = {
+    clearAll: () => clearFacet(),
```
Comment (fzaninotto) [inline src/app/js/public/facet/AppliedFacetList.js:53]: why use a different name?

File: src/app/js/public/graph/sagas.js [hunk 2]
```diff
@@ -76,15 +83,22 @@ export function* handleLoadChartDataRequest({ payload: { field } }) {
...
+    yield put(loadChartDataSuccess({ name, data: null }));
 }
 
 export default function*() {
     yield fork(function*() {
-        yield takeEvery(
-            [LOAD_CHART_DATA, TOGGLE_FACET_VALUE, CLEAR_FACET, APPLY_FILTER],
+        yield throttle(
+            500,
+            [
```
Comment (fzaninotto) [inline src/app/js/public/graph/sagas.js]: I can't find in the `redux-saga` doc the part about `throttle` taking an array as second argument. Can you add a link in a JS comment?

**Review by fzaninotto — CHANGES_REQUESTED** [review]

File: src/app/js/public/facet/AppliedFacetList.js (hunk 1 above)
Comment (ThieryMichel) [reply src/app/js/public/facet/AppliedFacetList.js:53]: to emphasize that this call will clear all facet and not just one

Comment (ThieryMichel) [reply src/app/js/public/facet/AppliedFacetList.js:53]: And also to not pass click event in the call.

**Review by fzaninotto — APPROVED** [review]

**Pull request merged**